admin-dev/themes/default/scss/admin-theme.scss

```diff
@@ -1,3 +1,27 @@
+/**
```

Comment:
I'm surprised so many SCSS files were missing the header, it makes sense that they contain it However we should check that after compilation the header is not duplicated multiple times from each source